renderer/src/components/ActivityLog.tsx

```diff
@@ -1,37 +1,72 @@
-import React, { useEffect, useState } from 'react'
-import { Activity } from '../../../main/typings'
-
-export const ActivityLog : React.FC = () => {
-  const [activities, setActivities] = useState<Activity[]>([])
-
-  useEffect(() => {
-    (async () => {
-      const newActivities = await window.electron.getAllActivities()
-      setActivities(newActivities)
-    })()
-  }, [])
-
-  useEffect(() => {
-    const unsubscribe = window.electron.onActivityLogged(allActivities => {
-      setActivities(allActivities)
-    })
-    return unsubscribe
-  }, [])
-
-  // TODO: add proper CSS styling :)
-  const style: React.CSSProperties = {
-    listStyle: 'none',
-    textAlign: 'left',
-    fontSize: '1rem',
-    fontFamily: 'monospace',
-    border: '1px solid white',
-    padding: '1em'
-  }
-
-  return <>
-    <h3>Activity Log</h3>
-    <ul style={style}>
-      {activities.map(activity => <li key={activity.id}>{activity.message}</li>).reverse()}
-    </ul>
-  </>
+import { FC } from 'react'
+import { ActivityEventMessage } from '../typings'
+import dayjs from 'dayjs'
+import { ReactComponent as WarningIcon } from '../assets/img/warning.svg'
+import { ReactComponent as JobIcon } from '../assets/img/job.svg'
+
+const ActivityLogItem: FC<ActivityEventMessage> = (activity) => {
+  const time = new Intl.DateTimeFormat(window.navigator.language, {
+    hour: 'numeric', minute: 'numeric', second: 'numeric'
+  }).format(new Date(activity.timestamp))
```

Comment:
```suggestion

const dateTimeFormat = new Intl.DateTimeFormat(window.navigator.language, {
  hour: 'numeric', minute: 'numeric', second: 'numeric'
})

const ActivityLogItem: FC<ActivityEventMessage> = (activity) => {
  const time = dateTimeFormat.format(new Date(activity.timestamp))
```

Otherwise one could think the formatter needs to be created dynamically